bundles/mapping/mapmodule/mapmodule.ol2.js

```diff
@@ -466,6 +466,8 @@ Oskari.clazz.define('Oskari.mapframework.ui.module.common.MapModule',
             olStyle.graphicWidth = size;
             olStyle.graphicHeight = size;
 
+            olStyle.fillOpacity = 0.8;
```

Comment:
This case set default opacity only. If current function styleDef parameter has styleDef.image.opacity then use this: https://github.com/kuosman/oskari-frontend/blob/65b94834de3bc5116bc18fbbeb337ee0d6f4f752/bundles/mapping/mapmodule/mapmodule.ol2.js#L487-L489